package.json

```
@@ -8,6 +8,7 @@
   "main": "dist/index.js",
   "module": "dist/index.modern.js",
   "source": "src/index.js",
+  "types": "src/types/index.d.ts",
```

you'll probably need to add this for it to work in the bundling

```suggestion
  "types": "dist/index.d.ts",
  "engines": {
    "node": ">=10"
  },
  "scripts": {
    "build": "microbundle-crl --no-compress --format modern,cjs && cp src/types/index.d.ts /dist",
    "start": "microbundle-crl watch --no-compress --format modern,cjs",
```